src/pages/SharePage/index.tsx

```diff
@@ -9,8 +10,18 @@ import {
 } from "../../components";
 import "./SharePage.css";
 
+interface FolderInfos {
+  owner: {
+    profileImageSource: string;
+    name: string;
+  };
+  name: string;
+  links: Link[];
+}
+
 const SharePage = () => {
-  const [folderData, setFolderData] = useState([]);
+  const [folderData, setFolderData] = useState<FolderInfos>({} as FolderInfos);
```

Comment:
```
{} as FolderInfos
```
웬만하면 `as`는 쓰지 않는것이 좋습니다.  folderData가 제대로 set되지 않을 경우 계속 빈 객체로 남게되는데 그럴 경우 에러가 납니다. 

차라리 기본값을 주는게 좋다고 생각합니다. 


```suggestion
const [folderData, setFolderData] = useState<FolderInfos>({
    links: [],
    name: "",
    owner: {
      name: "",
      profileImageSource: "",
    },
  });
```